contracts/Overtime/SportsAMMV2.sol

```diff
@@ -165,11 +172,21 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
             ISportsAMMV2RiskManager.RiskStatus riskStatus
         )
     {
-        (totalQuote, payout, fees, amountsToBuy, riskStatus) = _tradeQuote(_tradeData, _buyInAmount, true);
+        if (_collateral != address(0)) {
```

Comment:
should be skipped if Collateral is zero or if Collateral is the default collateral

Reply:
better to use require to ensure that collateral sent as param cannot be the defaultCollateral

Reply:
done